src/content/self-hosted/helm-configuration.mdx

```diff
@@ -437,6 +448,9 @@ The jobs that deploy your [development environments from Git](development/deploy
 - `extraEnv`: Environment variables to be set on the installer job containers.
 - `labels`: Labels to add to the installer job pods.
 - `runner`: to configure your [custom installer image](admin/custom-installer-image.mdx).
+  - `registry`: The registry where the installer image is hosted. Defaults to `` (You can change all the registries by setting `globals.registry` to your registry).
```

Comment:
Is the default here supposed to be blank?

Reply:
I would say "Defaults to the helm configuration setting `globals.registry`"